index.html

```diff
@@ -15,10 +15,13 @@
       <nav class="toolbar">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_583](https://user-images.githubusercontent.com/69638013/210242699-293a2c41-cde3-4ff2-9d1a-687c63edd421.png) |  ![Screenshot_582](https://user-images.githubusercontent.com/69638013/210242349-e008d20b-a82e-42cf-98ba-37febc16601a.png)


- [ ] Kindly make sure the close button is visible on the mobile menu, please remove the white background from the mobile menu 
- [ ] Please align the  mobile links and close icon to look like the  Figma design,
- [ ] Please use the same size used in the design for the close icon

- All these adjustments will make your design look like the Figma design,  see the video below

https://user-images.githubusercontent.com/69638013/210243471-f7bfb363-161b-4e84-ae49-48384ff3b82a.mp4